smac/acquisition/maximizer/helpers.py

```diff
@@ -72,44 +72,3 @@ def __len__(self) -> int:
             self._challengers = self._challengers_callback()
 
         return len(self._challengers) - self._index
-
-
-'''
-class FixedSet(AbstractAcquisitionMaximizer):
```

Comment:
What is that? 😅

Reply:
I don't know, and since it is commented out completely, it is not used anyway, and while I was there, I thought it best to remove it then.